funsite/templates/lms/register.html

```diff
@@ -23,9 +23,11 @@
                      error: onError});
          });
 
-         function onSuccess(json) {
-             var url = json.redirect_url || "${reverse('dashboard')}";
-             location.href = url;
+         function onSuccess() {
+            var queryParameters = new URLSearchParams(location.search);
+            var next = queryParameters.get('next');
```

Comment:
what if 'next' is missing ? what would be the default value ?

Reply:
Good catch, added in last fixup.

Reply:
nice :heavy_check_mark: :rocket: